File: 2037-count-square-sum-triples/2037-count-square-sum-triples.py
class Solution:
    def countTriples(self, n: int) -> int:
        c=0
        for i in range(1,n):
            for j in range(1,n):
                if(i!=j):
                   k=(((i*i)+(j*j))**(0.5))
                   if(k == int(k)):
                    if(k<=n):
                        c+=1  
        return c

File: 2037-count-square-sum-triples/2037-count-square-sum-triples.cpp
class Solution {
public:
    int countTriples(int n) {
        int c=0;
        for (int i=1;i<n;i++){
            for (int j=1;j<n;j++){
                if(i!=j){
                    double k= sqrt((i*i)+(j*j));
                    if(k==(int)(k)){
                        if(k<=n){
                            c++;
                        }
                    }
                }
            }
        }
        return c;
    }
};